### ISSUE TITLE
is there any breaking change from v4 to v5?

### ISSUE THREAD

**Issue by katlimruiz** [jeffijoe/awilix#267]
I have to ask here since I can't find any release note that mentions this.

Thanks!

**Comment by jeffijoe** [comment]
Yes, they are listed in the [changelog](https://github.com/jeffijoe/awilix/blob/master/CHANGELOG.md).
